app/controllers/dashboards_controller.rb

```diff
@@ -23,6 +21,7 @@ def build
     end
 
     redirect_to dashboards_path
+    `rake fix_coord_mismatch` unless ENV.fetch('RAILS_ENV') == 'test'
```

Comment:
Let's not skip code if we're in the test env. The point of tests is to make sure our code works. This is removing code coverage.

Reply:
removed ENV.fetch('RAILS_ENV') == 'test'